### PR TITLE
Update .gitignore, and fix missing dependency

### PR DESCRIPTION
**Pull request by CopyDemon**
Fixes
1. Remove `lib/` from .gitignore for now, to make sure when clone this repo, all dependency is there. (in future move dependency to npm package.json then won't need to keep all dependency files in repo).

2. Add `.DS_Store` to .gitignore for Mac developer.

3. Copy lib folder from `idaes-pse/idaes/core/ui/fsvis/static` to `idaes-ui/idaes_ui/fv/static/` to fix missing dependency issue.

Proposed changes:

Frontend won't run properly because of missing dependency.

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

**Comment by CopyDemon** [comment]
Currently, this UI repo is running ok by applying fixs above, but still has a warning related to pyomo:
```
pyomo/contrib/pynumero/sparse/block_matrix.py:25: 

DeprecationWarning: 
Please use `get_index_dtype` from the `scipy.sparse` namespace, the `scipy.sparse.sputils` namespace is deprecated.
from scipy.sparse.sputils import get_index_dtype
```

File: pyproject.toml [hunk 1]
```diff
@@ -43,7 +43,9 @@ build-backend = "setuptools.build_meta"
         # For running tests
         "pytest",
         # IDAES release
-        "idaes-pse"
+        "idaes-pse",
+        # install module requests, without this CICD on github has a waring: 'requests module not found'
```
Comment (dangunter) [inline pyproject.toml]: typo 'waring'

**Review by dangunter — CHANGES_REQUESTED** [review]
minor typo - otherwise looks good

File: pyproject.toml (hunk 1 above)
Comment (CopyDemon) [reply pyproject.toml]: Changes applied

**Pull request merged**